receiver/promtailreceiver/factory.go

```diff
@@ -0,0 +1,48 @@
+package promtailreceiver
```

Comment:
Please add the license header to all go files.

Reply:
done